radixdlt/src/main/java/com/radixdlt/network/EventCoordinatorNetworkRx.java

```diff
@@ -15,14 +15,18 @@
  * language governing permissions and limitations under the License.
  */
 
-package com.radixdlt.consensus;
+package com.radixdlt.network;
+
+import com.radixdlt.consensus.NewRound;
+import com.radixdlt.consensus.Vertex;
+import com.radixdlt.consensus.Vote;
 
 import io.reactivex.rxjava3.core.Observable;
 
 /**
  * Network accessor for the EventCoordinator
  */
-public interface NetworkRx {
+public interface EventCoordinatorNetworkRx {
```

Comment:
Hmm the name `EventCoordinatorNetworkRx` is not great IMHO. I would drop the`Rx` suffix all together (everywhere), since that really is an implementation detail. Might makes sense to keep for the purpose of disambiguation - if we have a non RX variant. But we dont? 

But also the name does not make lot of sense seeing it. Should this `interface` be renamed `NetworkController` ? And then an implementation ought to be nested (static class), i.e. `EventCoordinator.NetworkController`, where EventCoordinator is a class (or `EventCoordinatorImpl` ?) .

That makes much more sense to me :)

Reply:
I think the idea from @talekhinezh was that "rx" was used as a (fairly standard) abbrev for "receive".